script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_2.txt

```diff
@@ -91,7 +91,7 @@ Yumizuka runs, scampering to her seat.
 <Page27>
 The bell rings.
 "........."
-I'm really happy, but I can't be honest about it.
+I'm really happy, but I can't openly display it.
```

Comment:
Not sure about this - I don't think it's that he can't let people see he's happy, but that he is internally conflicted. Suggest something along these lines - 
> 本気で嬉しいのだけど、素直に喜べないこの複雑さ。
> I'm really happy, but there's a knot in the pit of my stomach. 
> I'm overjoyed, but I can't accept these feelings so easily

Reply:
second one works for me